app/components/visualizer/main-viz/templates/chart.jsx

```diff
@@ -0,0 +1,48 @@
+import D3Chart from "./D3Chart.js"
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+    this.chart = new D3Chart(el, {
+      width: "100%",
```

Comment:
So I started properly doing it... I had to redo approx 20 commits and I finally quit.  I then skipped through the next 30 commits, but it left me with a totally f'ed up branch.